pandas/core/generic.py

```diff
@@ -5201,60 +5201,209 @@ def abs(self):
         """
         return np.abs(self)
 
```

Comment:
The main reason I moved it out of the ``_shared_docs`` is that after adding doctests to the docstring I got this error when I tried to make the documentation. I came to the same conclusion as @jorisvandenbossche that the feature was unnecessary so I decided to dodge the bug by moving its location down to the class' docstring.

```python
 % python make.py
Converting source/html-styling.ipynb
No module named jupyter_client.manager
Failed to convert source/html-styling.ipynb
Running Sphinx v1.5.1
Exception occurred while building, starting debugger:
Traceback (most recent call last):
  File "/home/palewire/.virtualenvs/pandas/local/lib/python2.7/site-packages/sphinx/cmdline.py", line 295, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/home/palewire/.virtualenvs/pandas/local/lib/python2.7/site-packages/sphinx/application.py", line 163, in __init__
    confoverrides or {}, self.tags)
  File "/home/palewire/.virtualenvs/pandas/local/lib/python2.7/site-packages/sphinx/config.py", line 134, in __init__
    execfile_(filename, config)
  File "/home/palewire/.virtualenvs/pandas/local/lib/python2.7/site-packages/sphinx/util/pycompat.py", line 129, in execfile_
    exec_(code, _globals)
  File "/home/palewire/.virtualenvs/pandas/local/lib/python2.7/site-packages/six.py", line 699, in exec_
    exec("""exec _code_ in _globs_, _locs_""")
  File "<string>", line 1, in <module>
  File "conf.py", line 17, in <module>
  File "/home/palewire/Code/pandas/pandas/__init__.py", line 39, in <module>
    from pandas.core.api import *
  File "/home/palewire/Code/pandas/pandas/core/api.py", line 10, in <module>
    from pandas.core.groupby import Grouper
  File "/home/palewire/Code/pandas/pandas/core/groupby.py", line 40, in <module>
    from pandas.core.frame import DataFrame
  File "/home/palewire/Code/pandas/pandas/core/frame.py", line 65, in <module>
    from pandas.core.generic import NDFrame, _shared_docs
  File "/home/palewire/Code/pandas/pandas/core/generic.py", line 94, in <module>
    class NDFrame(PandasObject):
  File "/home/palewire/Code/pandas/pandas/core/generic.py", line 5271, in NDFrame
    @Appender(_shared_docs['describe'] % _shared_doc_kwargs)
TypeError: not enough arguments for format string
> /home/palewire/Code/pandas/pandas/core/generic.py(5271)NDFrame()
-> @Appender(_shared_docs['describe'] % _shared_doc_kwargs)
```